src/components/templates/descopeMiddleware.tsx

```diff
@@ -130,6 +130,24 @@ export const DescopeMiddleware = ({ children }: { children: ReactNode }) => {
 		setDescopeRenderKey((prevKey) => prevKey + 1);
 	};
 
+	const handleCliLoginRedirect = (): boolean => {
+		const redirPath = Cookies.get(systemCookies.redir);
+
+		if (redirPath) {
```

Comment:
Please reverse the condition, meaning:

```suggestion
		if (!redirPath) return false;
```